service-loadbalancer/Makefile

```diff
@@ -16,10 +16,13 @@ push: container
 
 haproxy:
 	docker build -t $(HAPROXY_IMAGE):$(TAG) build
-	docker create --name $(HAPROXY_IMAGE) $(HAPROXY_IMAGE):$(TAG) true
-	docker cp $(HAPROXY_IMAGE):/work/x86_64/haproxy-1.6-r0.apk haproxy.apk
+	docker create --name $(HAPROXY_IMAGE) $(HAPROXY_IMAGE):$(TAG) true	
+	docker cp $(HAPROXY_IMAGE):/work/x86_64/haproxy-1.6-r0.apk .
```

Comment:
please add a oneline comment, something like:
// docker cp semantics changed between 1.7 and 1.8, so we cp the file to cwd and rename it.